channeldb/channel.go

```diff
@@ -1756,11 +1768,17 @@ func (c *OpenChannel) AppendRemoteCommitChain(diff *CommitDiff) error {
 // this new pending commitment. Once they revoked their prior state, we'll swap
 // these pointers, causing the tip and the tail to point to the same entry.
 func (c *OpenChannel) RemoteCommitChainTip() (*CommitDiff, error) {
+	return c.RemoteCommitChainTipFrom(false)
+}
+
+// RemoteCommitChainTip returns the "tip" of the current remote commitment
+// chain. The confirmed bool indicates which bucket to retrieve the data from.
+func (c *OpenChannel) RemoteCommitChainTipFrom(confirmed bool) (*CommitDiff,
```

Comment:
If we do https://github.com/lightningnetwork/lnd/pull/3952, the `...From()` isn't needed.